index.js

```diff
@@ -8,16 +8,19 @@ var debug = require('debug')('detective-sass');
  * @param  {String} fileContent
  * @return {String[]}
  */
-module.exports = function detective(fileContent) {
+module.exports = function detective(fileContent, opts) {
   if (typeof fileContent === 'undefined') { throw new Error('content not given'); }
   if (typeof fileContent !== 'string') { throw new Error('content is not a string'); }
+  if (Object.prototype.toString.call(opts) !== '[object Object]') { opts = {}; }
```

Comment:
Interesting choice here. I would have gone with defaulting `opts` if it was undefined, but you're also doing a type check while defaulting. Cool!